log4j-core-test/src/test/java/org/apache/logging/log4j/core/impl/NestedLoggingFromThrowableMessageTest.java

```diff
@@ -80,14 +80,15 @@ public void testNestedLoggingInLastArgument() throws Exception {
         final Set<String> lines2 = readUniqueLines(file2);
 
         assertEquals("Expected the same data from both appenders", lines1, lines2);
-        assertEquals(3, lines1.size());
+        assertEquals(2, lines1.size());
         assertTrue(lines1.contains("INFO NestedLoggingFromThrowableMessageTest Logging in getMessage "));
         assertTrue(lines1.contains("ERROR NestedLoggingFromThrowableMessageTest Test message"));
```

Comment:
Not really, since lines 2 and 3 get checked against their start, not the complete content. Nevertheless, the situation is improved in 2022db7d3910bcd07b006452efc12e99482ca14c.